lib/global_id/locator.rb

```diff
@@ -106,7 +106,9 @@ def use(app, locator = nil, &locator_block)
 
       private
         def locator_for(gid)
-          @locators.fetch(normalize_app(gid.app)) { default_locator }
+          @locators.fetch(normalize_app(gid.app)) do
+            gid.respond_to?(:unscoped) ? ACTIVE_RECORD_LOCATOR : DEFAULT_LOCATOR
```

Comment:
seems a bit odd that if responds to :unscoped it means it's an AR. I would rather use `defined?(ActiveRecord::Base)` and `gid < ActiveRecord::Base`